### PR TITLE
 [CQ-4349416] fix wizard issue

### PR DESCRIPTION
**Pull request by amit-mnnit**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Review by nit23uec — APPROVED** [review]

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 }
 
+.cmp-adaptiveform-dropdown__widget[multiple='multiple'] {
```
Comment (nit23uec) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css:25]: I'd prefer it being part of the theme. we can fix this in the next release.

Comment (nit23uec) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css:25]: lets remove the height from this css.

Comment (amit-mnnit) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css:25]: yes

Comment (amit-mnnit) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css:25]: This is the default height of multi-select like textarea. Users can also override it through the theme.

Comment (amit-mnnit) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css]: removed

**Pull request merged**